tests/platform_tests/test_link_down.py

```diff
@@ -158,20 +164,33 @@ def test_link_down_on_sup_reboot(duthosts, localhost, enum_supervisor_dut_hostna
     # RP doesn't have any interfaces, check all LCs' interfaces
     link_status_on_all_LC(duthosts, localhost, fanouts_and_ports, up=False)
 
-    time.sleep(MAX_TIME_TO_REBOOT)
```

Comment:
can you add `time.sleep(MAX_TIME_TO_REBOOT)` at the beginning of test?
There are some errors due to reboot happened before this test file for some reason, and SUP may not have enough time to recover all dockers
and the wait for process wait for 300 sec, which would not be enough:
https://github.com/sonic-net/sonic-mgmt/blob/387f166751ee23205e46b28eccc0bad7d6dc1724/tests/common/platform/processes_utils.py#L50